CHANGELOG.md

```diff
@@ -3,6 +3,12 @@ Changelog
 
 All notable changes to this project will be documented in this file.
 
+## 4.41.0 - TBD
+
+### Changed
+
+- The `-cgo` suffixed docker images are no longer built and pushed along with the regular images. This decision was made due to low demand, and the unacceptable cadence with which the image base (Debian) receives security updates. It is still possible to create your own builds using the same methods as before.
```

Comment:
Do we want to specifically add the build command for users? Just to remind them they need to add the tag?

Reply:
Yeah that makes sense, added.